src/darsia/image/patches.py

```diff
@@ -637,10 +633,7 @@ def blend_and_assemble(self, update_img: bool = False) -> darsia.Image:
         # Define resulting darsia image
         da_assembled_img = darsia.Image(
```

Comment:
```suggestion
        da_assembled_img = type(self.base)(
```
I would also suggest to return the same type as the original image. So if an `OpticalImage` e.g. is provided as input (which knows about `color_space` while the others do not), also the assembled image does. Could you include this change and similarly for the other creations of Images in the same file? Then this bug would be also corrected at the same time.

Reply:
This makes sense. Haven't thought of using it this way before... Very neat. I'll update it.